functions/newman/src/envConfig.js

```diff
@@ -0,0 +1,29 @@
+import constants from './constants';
+
+function configMissing(configKey) {
+  throw new Error(`missing required .env setting for ${configKey}`);
+}
+
+const requiredConfig = () => ({
+  that: {
+    messagingReadQueueRate:
+      process.env.MESSAGING.READ_QUEUE_RATE ||
+      constants.THAT.MESSAGING.MESSAGING.READ_QUEUE_RATE ||
+      configMissing('MESSAGING.READ_QUEUE_RATE'),
+    slackWebhookUrl: process.env.SLACK_WEBHOOK_URL,
+    slackChannelOrder: process.env.SLACK_CHANNEL_ORDER || '#orders',
+  },
+  postmark: {
+    apiToken:
+      process.env.POSTMARK_API_TOKEN || configMissing('POSTMARK_API_TOKEN'),
+    emailFrom: process.env.POSTMARK_EMAIL_FROM || 'hello@thatconference.com',
```

Comment:
we might want to swap this based on that.us or thatconference.com.. not really sure how we might do that. we could also alias off that.tech and send all transactional shit from that too.